crates/p2p_proto/src/transaction.rs

```diff
@@ -174,6 +174,13 @@ pub enum Transaction {
     L1HandlerV0(L1HandlerV0),
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, ToProtobuf, TryFromProtobuf, Dummy)]
+#[protobuf(name = "crate::proto::transaction::Transaction")]
+pub struct Transaction {
+    pub txn: TransactionEnum,
```

Comment:
```suggestion
    pub txn: TransactionVariant,
```
The core type uses the Variant suffix - so maybe let's make it the same?

Reply:
well, OK, although as a first time contributor I'd say the various classes are confusing enough even if they don't have the same names... :-)

Reply:
True, however I think that it's better to have the same/similar naming and use namespacing, so that later on you can follow a particular flow easier. There's a lot of types involved in p2p and imo it's easier to follow if their names are similar on every level. Otherwise we would have to resort to very creative naming and the number of type names would probably explode making navigation more difficult.